src/common/object-name.js

```diff
@@ -0,0 +1,14 @@
+import { connectStore } from 'utils'
+import { createGetObject } from 'selectors'
+import React, { Component } from 'react'
+
+@connectStore(() => {
+  const object = createGetObject()
+  return (state, props) => ({object: object(state, props)})
+})
+export default class ObjectName extends Component {
+  render () {
+    const { object } = this.props
+    return <span>{object && object.name_label}</span>
+  }
+}
```

Comment:
I am not sure I like this pattern…
I have no issue with it being used but exposed as a reusable component in `common/` I am not sure about it…